src/mon/MonCommands.h

```diff
@@ -458,6 +458,11 @@ COMMAND("mon set-rank " \
 	"name=rank,type=CephInt",
 	"set the rank for the specified mon",
 	"mon", "rw")
+COMMAND("mon set-addrs " \
+	"name=name,type=CephString " \
+	"name=addrs,type=CephString",
+	"set the addrs (IPs and ports) a specific monitor binds to",
+	"mon", "rw")
```

Comment:
If this command is just for testing, shall we set the `HIDDEN` flag?

Reply:
I think it's a useful (if low-level) command.  Without it you can't change a monitor's addr (port, ipv4 vs ipv6, etc) without removing and re-adding it.